hbase-protocol/src/main/protobuf/MultiRowMutation.proto

```diff
@@ -37,9 +37,11 @@ message MutateRowsRequest {
   optional uint64 nonce_group = 2;
   optional uint64 nonce = 3;
   optional RegionSpecifier region = 4;
+  repeated Condition condition = 5;
 }
 
 message MutateRowsResponse {
+  optional bool processed = 1;
```

Comment:
Is this a backward-incompatible breaking change to the wire format? How does a client generated against the old definition handle the presence of any fields?